kotlin/vector-tiled-layer-custom-style/README.md

```diff
@@ -0,0 +1,44 @@
+# ArcGIS vector tiled layer (custom style)
+
+Load ArcGIS vector tiled layers using custom styles.
+
+Custom styled ArcGIS vector tiled layer             |  ArcGIS vector tiled layer list             |  Offline custom style
+:-------------------------:|:-------------------------:|:-------------------------:
+![Custom styled ArcGIS vector tiled layer](vector-tiled-layer-style.png)  |  ![ArcGIS vector tiled layer list](vector-tiled-layers-list.png)  |  ![Offline custom style](vector-tiled-layer-custom-2.png)
```

Comment:
I know the common design has two screenshots, but we don't support more than one one in the android sample viewer. Just use the first. It also needs to match the title like:

![Image of ArcGIS vector tiled layer custom style]arcgis-vector-tiled-layer-custom-style.png